File: helloworld_web_war/src/main/java/org/example/demo/DemoApp.java
package org.example.demo;

import org.noear.solon.Solon;
import org.noear.solon.annotation.SolonMain;

/**
 * @author noear
 * @since demo 1.0
 */
@SolonMain
public class DemoApp {
    public static void main(String[] args) {
        Solon.start(DemoApp.class, args);
    }
}

File: helloworld_web_jar_kt/src/main/kotlin/com/example/demo/DemoApp.kt
package com.example.demo

import org.noear.solon.Solon
import org.noear.solon.annotation.SolonMain

@SolonMain
class App {
    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            Solon.start(App::class.java, args)
        }
    }
}